tools/image_processing/imagej2/imagej2_convert_format/imagej2_convert_format.xml

```diff
@@ -36,24 +36,22 @@
     </outputs>
     <tests>
         <test>
-            <param name="input" value="dot_blot.jpg" ftype="jpg" />
-            <param name="max_heap_size_type" value="megabytes" />
-            <param name="max_heap_size" value="256" />
+            <param name="input" value="dot_blot.jpg" />
             <param name="input_datatype" value="jpg" />
             <param name="output_datatype" value="png" />
-            <output name="output" file="dot_blot.png" ftype="png" />
```

Comment:
@bgruening This is what I thought, but it seems only some tests will pass if image types are used in tags.  If I remember correctly, gif and tiff do not work and I think there are others.  Maybe my configuration is wrong?